src/tests/ftest/dfuse/bash.py

```diff
@@ -119,6 +119,9 @@ def run_bashcmd(self, il_lib=None, compatible_mode=False):
                             f'realpath {link_name}',
                             f'head {fuse_root_dir}/src.c',
                             f'tail {fuse_root_dir}/src.c',
+                            'printf "exec 5>config.log\n{\ncat <<EOF\n------\nEOF\n} >&5\n"'
+                            f'> {fuse_root_dir}/1.sh; cd {fuse_root_dir}; '
+                            f'/usr/bin/sh {fuse_root_dir}/1.sh',
```

Comment:
I assume this is covered in https://github.com/daos-stack/daos/pull/13681 in which case please drop the changes to this file.

Reply:
ok. I will drop the changes here. Thank you!